pkg/daemon/update_test.go

```diff
@@ -64,7 +64,7 @@ func TestUpdateOS(t *testing.T) {
 
 // TestReconcilable attempts to verify the conditions in which configs would and would not be
 // reconcilable. Welcome to the longest unittest you've ever read.
-func TestReconcilable(t *testing.T) {
+func Testreconcilable(t *testing.T) {
```

Comment:
nit: You might want to leave this func and `testmachineConfigDiff` in upper case (if not, this func's comment should be adapted)